beat/apps/importFaultShp.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jun  1 18:22:36 2021
+
+@author: Marin Govorcin
+"""
+
+import numpy as np
+import os, sys
+import argparse
+
+class faultSegments:
+    def __init__(self, segmentPointsLonLat= None, segmentPointsXY= None, segmentMidPoints= None, segmentStrike= None, segmentLength= None):
+        if segmentPointsLonLat is None:
+            self.segmentPointsLonLat = []
+        else:
+            self.segmentPointsLonLat = segmentPointsLonLat
+        if segmentPointsXY is None:
+            self.segmentPointsXY = []
+        else:
+            self.segmentPointsXY = segmentPointsXY
+        if segmentMidPoints is None:
+            self.segmentMidPoints = []
+        else:
+            self.segmentMidPoints = segmentMidPoints
+        if segmentStrike is None:
+            self.segmentStrike = []
+        else:
+            self.segmentStrike = segmentStrike
+        if segmentLength is None:
+            self.segmentLength = []
+        else:
+            self.segmentLength = segmentLength
+            
+    def addPointsLonLat(self,data):
+        self.segmentPointsLonLat.append(data)
+           
+    def addPointsXY(self,data):
+        self.segmentPointsXY.append(data)
+                  
+    def addMidPoints(self,data):
+        self.segmentMidPoints.append(data)
+    
+    def addStrikes(self,data):
+        self.segmentStrike.append(data)
+    
+    def addLengths(self,data):
+        self.segmentLength.append(data)
+    
+    def plotFault(self):
+        from matplotlib import pyplot as plt
+        
+        #plot fault segments
+        z=1
+        for i in range(len(self.segmentPointsXY)):
+            for j in range(len(self.segmentPointsXY[i])):
+                plt.plot(self.segmentPointsXY[i][j,:].reshape(2,2)[0],self.segmentPointsXY[i][j,:].reshape(2,2)[1],'-k')
+                plt.plot(self.segmentMidPoints[i][j,0],self.segmentMidPoints[i][j,1],'bo')
+                plt.text(self.segmentMidPoints[i][j,0]+100,self.segmentMidPoints[i][j,1]+100, 'F{} , Strike: '.format(z) + str(int(self.segmentStrike[i][j]))+'°')
+                z = z +1 
+        #plot event location
+        plt.plot(0,0,'r+', label = 'Event location',ms=14)
+        plt.xlabel("East shift [m]")
+        plt.ylabel("North shift [m]")
+        plt.title(" Fault segements to model in BEAT")
+        
+
+def import_faultShp(daShapefile):
```

Comment:
need documentation somewhere how the shapefile needs to be structured to be readable with this script.